coordinator/performance/monitor.go

```diff
@@ -101,7 +101,6 @@ func (m *Monitor) runQueries() Status {
 				mlog.String("query_threshold", fmt.Sprintf("%2.8f", query.Threshold)),
 			)
 			status = Status{Alert: true}
-			break
```

Comment:
This change is unrelated, but while reviewing this logic, I thought it would be better to alert about (and log) all the queries that hit the threshold rather than the first one only.